search_username/catalog/views.py

```diff
@@ -0,0 +1,77 @@
+import datetime
+import itertools
+
+from .models import SaveUsers, Base
+
+from django.shortcuts import render
+
+from InstagramAPI.InstagramAPI import InstagramAPI
+
+from search_username.settings import password, user_name
+
+from sqlalchemy import create_engine
```

Comment:
Third party dependencies should in one group of imports, your own - another group, system libraries - third group.

Reply:
done